docs/src/manual/variables.md

```diff
@@ -1495,23 +1495,95 @@ The most important implication of this design is that JuMP treats a parameter
 multiplied by a decision variable as a quadratic expression, even though it is
 equivalent to a linear expression.
 
-```jldoctest nonlinear_parameters
-julia> model = Model();
+```jldoctest
+julia> begin
+           model = Model()
+           @variable(model, x >= 3)
+           @variable(model, p in Parameter(2))
+           @objective(model, Min, p * x)
+           objective_function_type(model)
+       end
+QuadExpr (alias for GenericQuadExpr{Float64, GenericVariableRef{Float64}})
+```
 
-julia> @variable(model, x);
+As a consequence, solving a "linear" program with a solver like HiGHS fails:
+```jldoctest
+julia> using HiGHS
+
+julia> begin
+           model = Model(HiGHS.Optimizer)
+           set_silent(model)
+           @variable(model, x >= 3)
+           @variable(model, p in Parameter(2))
+           @objective(model, Min, p * x)
+           optimize!(model)
+           is_solved_and_feasible(model)
+       end
+false
+```
+because the model is equivalent to a quadratic program with an indefinite
+objective:
```

Comment:
Indefined might be ambiguous.
Maybe non-positive semidefinite?

Reply:
If a matrix has both positive and negative Eigen values isn't the correct term indefinite?